style.css

```diff
@@ -1 +1,94 @@
-/* 본인의 디자인 감각을 최대한 발휘해주세요! */
+@import url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css");
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Pretendard";
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background: #F3FAFF;
+}
+
+.container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 50%;
+    min-width: 550px;
+    max-width: 650px;
+    margin: 0 auto;
+}
+
+.main {
+    text-align:left;
+    background: white;
+    padding: 7% 15% 7% 15%;
+    border-radius: 10px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    flex-grow: 1;
+    margin: 0 20px;
+}
```

Comment:
<img width="1460" alt="image" src="https://github.com/user-attachments/assets/2f90be51-5015-40c6-9724-18cf99c3af38">


이런 경우는 거의 없겠지만 todo가 일정 갯수가 넘어가면 화면 잘림 현상이 나타납니다! 아래와 같이 최대 높이를 설정하고 스크롤을 활성화 시키면 좋을 것 같아요

```suggestion
  max-height: 100vh;
  overflow-y: auto; 
 }
```